packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -56,6 +56,13 @@ export interface ISemanticColors {
    */
   bodyBackground: string;
 
+  /**
+   * The standout color for highlighted content backgrounds.
+   * This should use the neutral variant when on no emphasis.
+   * This should be a shade darker than the other emphases.
```

Comment:
"shade darker than bodyBackground"
and something about it being lighter in inverted themes